corehq/apps/accounting/tests/test_migrations.py

```diff
@@ -0,0 +1,140 @@
+import random
+from datetime import date, timedelta
+
+from django.test import TestCase
+
+from corehq.apps.accounting.tests import generator
+from corehq.apps.accounting.models import (
+    BillingAccount,
+    DefaultProductPlan,
+    SoftwarePlanEdition,
+    Subscription,
+)
+from corehq.apps.accounting.tasks import assign_explicit_community_subscriptions
+from corehq.apps.domain.models import Domain
+
+
+class TestExplicitCommunitySubscriptions(TestCase):
+
+    domain = None
+    from_date = None
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestExplicitCommunitySubscriptions, cls).setUpClass()
+
+        # TODO - remove once messy tests are cleaned up
+        for domain in Domain.get_all():
```

Comment:
There's the `delete_all_domains` method in `test_utils` which does a bulk delete which might be faster